### DISCUSSION TITLE
Are my cameras not supported?

### DISCUSSION THREAD

**Discussion by ghost** [JurajNyiri/HomeAssistant-Tapo-Control#783] [Q&A] [answered]
I cannot for the life of me get my cameras to connect and keep getting connection failed. I've tried both my KC410S and my EC71 and both immediately give 'Connection failed' in the home assistant add integration screen. Are these models not supported? I could not find a supported model list anywhere.

**Comment by JurajNyiri** [comment]
All Tapo cameras are supported and the verified list is at https://github.com/JurajNyiri/HomeAssistant-Tapo-Control?tab=readme-ov-file#troubleshooting--faq .

Make sure HA can reach the camera under the IP and port 443 and others as specified in readme.

**Comment by JurajNyiri** [comment, marked as answer]
EC71 and KC410S are a Kasa camera, not Tapo.

**Reply by ghost** [reply]
Oh wow I'm silly thanks! Just assumed since my cameras synced with the Tapo app. Thanks for the quick reply!
